ext/hal/nordic/nrfx/hal/nrf_power.h

```diff
@@ -509,6 +509,25 @@ typedef enum
     NRF_POWER_MAINREGSTATUS_HIGH   = POWER_MAINREGSTATUS_MAINREGSTATUS_High    /**< High voltage mode. Voltage supplied on VDDH.  */
 }nrf_power_mainregstatus_t;
 
+/**
+ * @brief REG0 voltage definitions
```

Comment:
I don't think we can accept these changes, they should come through `nrfx` rather than being patched here.

Reply:
well, that's not really a "change", it's just some sort of mapping the config values to NRFX values

Reply:
Such mapping does not belong here (move it to _soc.c_ together with the function using it). The `NRF_POWER_` prefix is reserved for definitions directly related to the POWER module.